index.html

```diff
@@ -414,37 +404,50 @@ <h2>Attributes</h2>
               invoked.</p>
             </li>
             <li>
-              <p>Let <var>delay</var> be the argument to the setter.</p>
+              <p>Let <var>depth</var> be the argument to the setter.</p>
+            </li>
+            <li>
+              <p>If <var>depth</var> is negative or larger than 4000 milliseconds, then
+              [=exception/throw=] a {{RangeError}}.</p>
+            </li>
+            <li>
+              <p>Set <var>receiver</var>'s {{RTCRtpReceiver/[[JitterBufferDepth]]}}
+              to <var>depth</var>.</p>
             </li>
             <li>
-              <p>If <var>delay</var> is negative or larger than 4 seconds then,
-              [=exception/throw=] a {{RangeError}} and abort these steps.</p>
+              <p>If <var>depth</var> is below <a>minimum allowed
+              depth</a> or above <a>maximum allowed depth</a> then clamp
+              <var>depth</var> to <a>minimum allowed
+              depth</a> or <a>maximum allowed depth</a> to be as close as
+              possible to the original value.</p>
             </li>
             <li>
-              <p>Set the value of <var>receiver</var>'s {{RTCRtpReceiver/[[PlayoutDelay]]}}
-              internal slot to <var>delay</var>.</p>
+              <p>Let <var>track</var> be <var>receiver</var>'s
+              {{RTCRtpReceiver/[[ReceiverTrack]]}}.</p>
             </li>
             <li>
               <p>In parallel, begin executing the following steps:</p>
               <ol>
                 <li>
-                  <p>Update the underlying system about the new delay request,
-                  or that there is no hint if <var>delay</var> is
+                  <p>Update the underlying system about the new <var>depth</var>,
+                  or that there is no application preference if <var>depth</var> is
                   <code>null</code>.</p>
-                  <p>If the given delay value is below <a>minimum allowed
-                  delay</a> or above <a>maximum allowed delay</a> then the
-                  value used MUST be clamped to <a>minimum allowed
-                  delay</a> or <a>maximum allowed delay</a> to be as close as
-                  possible to the requested one.</p>
+                </li>
+                <p>
+                  If <var>track</var> is synchronized with another
+                  {{RTCRtpReceiver}}'s track for
+                  <a data-cite="RFC5888#section-7">audio/video synchronization</a>,
+                  then the <a>user agent</a> SHOULD use the larger of the two receivers'
+                  {{RTCRtpReceiver/[[JitterBufferDepth]]}} for both receivers.
```

Comment:
```suggestion
                  {{RTCRtpReceiver/[[JitterBufferDepth]]}} for both receivers.
                </p>
                <p>
                  When the underlying system is applying a jitter buffer depth, it will
                  continuously make sure that the actual jitter buffer depth is clamped
                  within the <a>minimum allowed depth</a> and <a>maximum allowed
                  depth</a>.
```